app/javascripts/main/spellcheckerManager.ts

```diff
@@ -104,102 +161,78 @@ export function createSpellcheckerManager(
     ko: '한국어' /** Korean */
   };
 
+  const availableSpellCheckerLanguages = Object.values(
+    Language
+  ).filter(language =>
+    session.availableSpellCheckerLanguages.includes(language)
+  );
+
+  if (
+    process.env.NODE_ENV === 'development' &&
```

Comment:
We should probably make some sort of helper func or const for `process.env.NODE_ENV === 'development'`. `isDev()` or something like that.